scripts/dev/build_release.py

```diff
@@ -123,6 +124,36 @@ def patch_mac_app():
         os.symlink(os.path.join(os.pardir, os.pardir, os.pardir, 'Contents',
                                 'MacOS', lib),
                    os.path.join(dest, lib))
+    # Patch Info.plist to declare URLs support
+    plist_path = os.path.join(app_path, 'Contents', 'Info.plist')
+    with open(plist_path, "rb") as f:
+        plist_data = plistlib.load(f)
+    plist_data.update(INFO_PLIST_UPDATES)
+    with open(plist_path, "wb") as f:
+        plistlib.dump(plist_data, f)
+
+
+INFO_PLIST_UPDATES = {
+    'CFBundleURLTypes': [{
+        "CFBundleURLName": "http(s) URL",
+        "CFBundleURLSchemes": ["http", "https"]
+    }, {
+        "CFBundleURLName": "local file URL",
+        "CFBundleURLSchemes": ["file"]
+    }],
+    'CFBundleDocumentTypes': [{
+        "CFBundleTypeExtensions": ["html", "htm"],
+        "CFBundleTypeMIMETypes": ["text/html"],
+        "CFBundleTypeName": "HTML document",
+        "CFBundleTypeOSTypes": ["HTML"],
+        "CFBundleTypeRole": "Viewer",
+    }, {
+        "CFBundleTypeExtensions": ["xhtml"],
+        "CFBundleTypeMIMETypes": ["text/xhtml"],
+        "CFBundleTypeName": "XHTML document",
+        "CFBundleTypeRole": "Viewer",
+    }]
+}
```

Comment:
`misc/qutebrowser.spec` also does some plist manipulations using the `info_plist` argument. Since PyInstaller can't do nested lists/dicts like we need here, I think it'd be best to move those here as well, so everything is in one place.

Reply:
Will do. Looks like I've found another plist issue. Those params (specified in qutebrowser.spec) are strings, but probably should be booleans:

```python
             info_plist={
                 'NSHighResolutionCapable': 'True',
                 'NSSupportsAutomaticGraphicsSwitching': 'True',
             },
```

I've checked the resulting Info.plist - they indeed produce string values, whilst Chrome's plist contains booleans. Should I fix that too?

Reply:
Was that with those in `qutebrowser.spec` and not with `plistlib`? I probably used strings because [the PyInstaller docs do so](https://pythonhosted.org/PyInstaller/spec-files.html#spec-file-options-for-a-mac-os-x-bundle), but sounds good to fix that as well while we're at it.

Reply:
@The-Compiler should I?

Reply:
> Was that with those in qutebrowser.spec and not with plistlib?

Just checked - it's the same for both variants. Looks like a documentation bug issue.